colossalai/booster/plugin/moe_hybrid_parallel_plugin.py

```diff
@@ -359,9 +359,6 @@ def configure(
                         max_norm=self.max_norm,
                         **self.amp_config,
                     )
-                    self.checkpoint_io.link_master_and_working_param(
```

Comment:
How to keep the consistency of fp16 params and fp32 master params when loading model?

Reply:
I align with hybrid parallel plugin in this change https://github.com/hpcaitech/ColossalAI/blob/c36e782d80bd3936fd3fec1f72dc0b476bf4016c/colossalai/booster/plugin/hybrid_parallel_plugin.py#L1093